debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -34,8 +34,10 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MongoDbConnectorConfig.class);
 
-    protected static final String COLLECTION_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG = "\"collection.include.list\" or \"collection.whitelist\" is already specified";
-    protected static final String DATABASE_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG = "\"database.include.list\" or \"database.whitelist\" is already specified";
+    protected static final String COLLECTION_EXCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG = "\"collection.exclude.list\" is already specified";
```

Comment:
When we have both `*.blacklist` and `*.exclude.list` are set we use the `*.exclude.list` value.
I think there shouldn't be an error on the `*.exclude.list` field, but and only on the `*.blacklist` field, as this value is discarded. WDYT?